db/migrate/20160429223134_remove_security_question_fields_from_users.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveSecurityQuestionFieldsFromUsers < ActiveRecord::Migration
+  def change
+    remove_column :users, :security_question_attempts_count
```

Comment:
That would have worked as well, but I prefer to use separate migrations for organization purposes, and to make it easier to roll back any changes separately. In this case, it probably doesn't matter, but I did it out of habit.